src/lib/createGHA/base.yml

```diff
@@ -0,0 +1,23 @@
+# This GitHub Action workflow was auto-generated by the `rdme` cli on :[](timestamp)
```

Comment:
We have this base file and we're using [`hercule`](npmjs.com/package/hercule) to populate it dynamically ✨

Reply:
(this is now outdated, see https://github.com/readmeio/rdme/commit/f03902b15b4bac663ddc8460af808f1f1e51ece9)